### PR TITLE
migrate repo verify to new cmds library.

### PR DESCRIPTION
**Pull request by Stebalien**
License: MIT
Signed-off-by: Steven Allen <steven@stebalien.com>

### CODE REVIEW

File: core/commands/repo.go [hunk 1]
```diff
@@ -298,63 +291,35 @@ var repoVerifyCmd = &oldcmds.Command{
...
-			}
-
-			obj, ok := v.(*VerifyProgress)
-			if !ok {
-				return nil, e.TypeErr(obj, v)
-			}
-
-			buf := new(bytes.Buffer)
-			if strings.Contains(obj.Msg, "was corrupt") {
-				fmt.Fprintln(os.Stdout, obj.Msg)
```
Comment (Stebalien) [inline core/commands/repo.go]: I removed this distinction. Was it really necessary? (also, I assume this was *supposed* to print to stderr...)

**Comment by whyrusleeping** [comment]
Do you intend for this to come before #4751?

**Comment by Stebalien** [comment]
I don't see how they're related. Does that PR touch repo verify? I'm trying to bypass an issue I've noticed in the cmds lib backwards compatibility layer by simply migrating away from it as fast as possible (the `expected <-chan interface{}, got <nil>` on `^C` error).

File: core/commands/repo.go (hunk 1 above)
Comment (Kubuxu) [reply core/commands/repo.go]: It was probably to be able to pipe just the corrupted blocks out into other command.

File: core/commands/repo.go [hunk 2]
```diff
@@ -298,63 +291,35 @@ var repoVerifyCmd = &oldcmds.Command{
...
-				return buf, nil
-			}
-
+	Encoders: cmds.EncoderMap{
+		cmds.Text: cmds.MakeTypedEncoder(func(req *cmds.Request, w io.Writer, obj *VerifyProgress) error {
```
Comment (Stebalien) [inline core/commands/repo.go]: Wait for https://github.com/ipfs/go-ipfs-cmds/issues/65 to be fixed before merging.

File: core/commands/repo.go (hunk 1 above)
Comment (Stebalien) [reply core/commands/repo.go]: I'd guess this was mean to write to stderr as the normal output is written to stdout.

File: core/commands/repo.go (hunk 2 above)
Comment (dignifiedquire) [reply core/commands/repo.go]: @stebalien the linked issue has since been fixed, was this the only blocker for moving this forward?

**Comment by Stebalien** [comment]
This was actually replaced by a newer PR (one rebased over a PR that parallelized the verification). Thanks for bumping this.

**Pull request closed without merging**